jsx/src/components/ServerDashboard/ServerDashboard.jsx

```diff
@@ -590,8 +584,13 @@ const ServerDashboard = (props) => {
           limit={limit}
           visible={user_data.length}
           total={total}
-          next={() => setOffset(offset + limit)}
-          prev={() => setOffset(offset - limit)}
+          // don't trigger via setOffset state change,
```

Comment:
Possible cleanup action here mentioning `setOffset` depending if it was removed.